### ISSUE TITLE
Add a "Keyboard shortcuts" action in the UI-Editor to list available shortcuts

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#2026]
### Issue description

The issue raised by @yuri1969 is to offer some helper action providing info about available shortcuts from the UI.

We would need some help here @Nico-Kestra ;) Some inspirations for you (to communicate what we mean):

![image](https://github.com/kestra-io/kestra/assets/86264395/e70b09fa-2aa6-41da-b11b-1aa6bc81cff3)

![image](https://github.com/kestra-io/kestra/assets/86264395/fd7db956-b312-487f-be02-ee1052e331c4)

**Comment by Nico-Kestra** [comment]
@anna-geller , do you have Kestra's shortcut?

**Comment by anna-geller** [comment]
These were mentioned by Ben in the related issue:

- move line ( already by using ALT + Up/Down Arrow )
- duplicate line ( already by using SHIFT + ALT + Up/Down Arrow )
- save flow ( already by using CTRL + s )
- comment/uncomment ( already by using CTRL + / or CTRL + k + c or CTRL + k + u )
- fold/unfold code ( already by using CTRL + k + l )

regarding placement of this, WDYT about adding it as a keyboard icon next to the editor? just an idea -- no preference on where to put it :)

![image](https://github.com/kestra-io/kestra/assets/86264395/4cbaa8ef-98ca-4a44-b995-08b58de25a5d)

**Comment by Nico-Kestra** [comment]
Oh, is it also for the editor? ok

**Comment by anna-geller** [comment]
yup, it's only for the Editor 👍 sorry, I should have clarified in the issue name. changing now

**Comment by Nico-Kestra** [comment]
We can use this icon
<img width="366" alt="Capture d’écran 2023-09-05 à 12 41 50" src="https://github.com/kestra-io/kestra/assets/125994028/38a21840-8313-41da-b8b3-933a60992c41">

And display this popin for shortcuts keyboard
<img width="842" alt="Capture d’écran 2023-09-05 à 12 41 54" src="https://github.com/kestra-io/kestra/assets/125994028/8e26ae0f-233f-483e-8f0b-2745cc7e1aa5">

**Comment by anna-geller** [comment]
@Nico-Kestra can we change:
- "Ctrl" to "Ctrl/Cmd" 
- "Alt" to "Alt/Option"

to include Mac users?

**Comment by anna-geller** [comment]
just a note that for now this is available as right click 
![image](https://github.com/kestra-io/kestra/assets/86264395/783acf56-3417-4f8b-bec1-993de371bddb)

**Comment by MilosPaunovic** [comment]
A question for @Ben8t or anybody on the product side - we have a PR opened for this issue by @Piyush-r-bhaskar (https://github.com/kestra-io/kestra/pull/6628), we just need confirmation that the icon for toggling the shortcuts is still needed on the place requested in the issue or do we want to place it somewhere else, as the issue is created a while back and some things changed in the meantime.

Current solution in the PR is this:
https://github.com/user-attachments/assets/e1d9b8ed-1e66-4da8-90ad-86af6d7cb10f

**Comment by Ben8t** [comment]
Good catch! It will be a great addition - The pop up is very nice ! however the position is not right. It's mixed with editor views.

@Nico-Kestra do you have any views on that. Could be part of the discussion we have on editor/views design

@MilosPaunovic let's wait Nico's input on that. I'm moving the issue to 0.22 as it's simple addition but very nice UX-wise (shortcut are very important in a developer workflow, and we don't highlight them enough IMO)

**Comment by MilosPaunovic** [comment]
Agreed @Ben8t, and the proposed solution from @Piyush-r-bhaskar does look very good in use, also.

So, Piyush the plan is to wait for input from Nico, and then just move the button position to where he decides; the logic and all the other things we talked about in the `PR` stay the same.

**Comment by Piyush-r-bhaskar** [comment]
> Agreed @Ben8t, and the proposed solution from @Piyush-r-bhaskar does look very good in use, also.
> 
> So, Piyush the plan is to wait for input from Nico, and then just move the button position to where he decides; the logic and all the other things we talked about in the `PR` stay the same.

Perfect. Sure @MilosPaunovic

**Comment by Nico-Kestra** [comment]
At the top right of the code area?

<img width="824" alt="Image" src="https://github.com/user-attachments/assets/dd5a9f62-2a5b-4cbd-9c92-56a804c6a33b" />

**Comment by MilosPaunovic** [comment]
Yeah, seems quite alright with me. @Piyush-r-bhaskar Could you do the necessary change?

**Comment by Piyush-r-bhaskar** [comment]
> Yeah, seems quite alright with me. [@Piyush-r-bhaskar](https://github.com/Piyush-r-bhaskar) Could you do the necessary change?

I will do it.

**Comment by MilosPaunovic** [comment]
@Piyush-r-bhaskar This is some great work here, thanks a lot for this!

@Nico-Kestra Once this is deployed, could you give it a final look and see if we need to amend coloring of it to align with the new scheme? If necessary, could you open a follow-up issue?
